extension/js/common/browser/browser.ts

```diff
@@ -16,6 +16,10 @@ export class Browser {
     return URL.createObjectURL(new Blob([content], { type: 'application/octet-stream' }));
   };
 
+  public static isThunderbirdMail = () => {
+    return Catch.browser().name === 'thunderbird';
+  };
```

Comment:
let's move it to https://github.com/FlowCrypt/flowcrypt-browser/blob/master/extension/js/common/platform/catch.ts, we have similar `isFirefox` method there

Reply:
Updated - thank you!